src/stores/logFileStore.ts

```diff
@@ -157,6 +157,8 @@ const useLogFileStore = create<LogFileState>((set, get) => ({
                 title: "Action failed",
             });
             setUiState(UI_STATE.UNOPENED);
+            setFileName(LOG_FILE_STORE_DEFAULT.fileName);
+            setLogData(VIEW_STORE_DEFAULT.logData);
```

Comment:
```suggestion
```
This should be addressed in a separate PR. @coderabbitai create an issue to track this